Not compile with Clang 15: error invalid suffix on literal; C++11 requires a space between literal and identifier

When trying compile lensfun 0.3.3 with Clang 15.0.6 on OpenMandriva Cooker I see build error:

```
DEBUG util.py:443:  /builddir/build/BUILD/lensfun-0.3.3/libs/lensfun/cpuid.cpp:95:18: error: invalid suffix on literal; C++11 requires a space between literal and identifier [-Wreserved-user-defined-literal]
DEBUG util.py:443:          "push %%"R_BX"\n" \
DEBUG util.py:443:                   ^
DEBUG util.py:443:                    
DEBUG util.py:443:  /builddir/build/BUILD/lensfun-0.3.3/libs/lensfun/cpuid.cpp:97:17: error: invalid suffix on literal; C++11 requires a space between literal and identifier [-Wreserved-user-defined-literal]
DEBUG util.py:443:          "pop %%"R_BX"\n" \
DEBUG util.py:443:                  ^
DEBUG util.py:443:                   
DEBUG util.py:443:  2 errors generated.
```

Full build log: https://file-store.openmandriva.org/api/v1/file_stores/c9296ed4d37005ddb33f914e40e45ee4055e6870.log?show=true

Worth to add previous unstable 0.3.95 compiled fine.
As workaround I switched right now to GCC and all works fine, but we trying use Clang for all packages and it would be good to Lensfun to compile with Clang.

--- a/libs/lensfun/cpuid.cpp
+++ b/libs/lensfun/cpuid.cpp
@@ -92,9 +92,9 @@ guint _lf_detect_cpu_features ()
 {
 #define cpuid(cmd) \
     __asm volatile ( \
-        "push %%"R_BX"\n" \
+        "push %%" R_BX "\n" \
         "cpuid\n" \
-        "pop %%"R_BX"\n" \
+        "pop %%" R_BX "\n" \
        : "=a" (ax), "=c" (cx),  "=d" (dx) \
        : "0" (cmd))
 

--- a/tests/test_modifier.cpp
+++ b/tests/test_modifier.cpp
@@ -78,8 +78,8 @@ void test_mod_projection_center(lfFixture* lfFix, gconstpointer data)
 // check if output becomes NaN when processing geometry conversion
 void test_mod_projection_borders(lfFixture* lfFix, gconstpointer data)
 {
-    float in[2]  = {lfFix->img_width, lfFix->img_height};
-    float in2[2] = {(lfFix->img_width-1)/2, (lfFix->img_height-1)/2};
+    float in[2]  = {(float) lfFix->img_width, (float) lfFix->img_height};
+    float in2[2] = {(float) (lfFix->img_width-1)/2, (float) (lfFix->img_height-1)/2};
     float res[2] = {0, 0};
 
     lfLensType geom_types [] = {LF_RECTILINEAR, LF_PANORAMIC, LF_EQUIRECTANGULAR, LF_FISHEYE_STEREOGRAPHIC, LF_FISHEYE, LF_FISHEYE_EQUISOLID, LF_FISHEYE_ORTHOGRAPHIC, LF_FISHEYE_THOBY, LF_UNKNOWN};

